### PR TITLE
Dart Script for Generating SQL Imports for Annya Pali Texts

### PR DESCRIPTION
**Pull request by iulspop**
This is WIP. I have my WIP notes/documenation in `iuliu_test` dir, but I will reorganize it to fit with the current repo later.

Todos:
- [x] comparing the VRI XML for Majjhimanikāya Mūlapaṇṇāsapāḷi with current DB data
- [x] documenting VRI XML to current DB schema mapping
- [x] comparing the VRI XML for Niruttidipanipatha with current DB data (I want to analyze one current Annya text example too before proceeding)
- [ ] Read Anudīpanīpāṭha XML file, and manually write a partial SQL import and test to verify it works
- [ ] Write a Dart script which inputs an XML file and outputs SQL import for our current DB schema (with tests)

### CODE REVIEW

File: iuliu_test/VRI-XML-explanation.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+But the PTS edition of the Vinaya is five volumes, so the count starts at 1:
+
+```
+<pb ed="P" n="1.0001" />
+<pb ed="P" n="2.0001" />
+<pb ed="P" n="3.0001" />
+<pb ed="P" n="4.0001" />
+<pb ed="P" n="5.0001" />
+```
```
Comment (iulspop) [inline iuliu_test/VRI-XML-explanation.md:34]: I induced this explanation for the two parts of the page beginning number from looking at various examples, but I'm not sure if it's correct. Do you know what they mean @bksubhuti?

**Comment by bksubhuti** [comment]
our information is a little fragmented.. with an issue and the pr.. but ven pndazza says that vri uses this script to generate html from xml..  I have no idea what this does.  When you click on it, it will prompt for  a download immediately and then you can open in a text editor.
[[https://tipitaka.org/romn/cscd/tipitaka-latn.xsl](https://l.messenger.com/l.php?u=https%3A%2F%2Ftipitaka.org%2Fromn%2Fcscd%2Ftipitaka-latn.xsl&h=AT2MIRvbgvoqFG4L_-x5yEeeS06SQViA9suTLg2-iZkuzhlaAz82TK79dpUmfO0ZKG9m7r-Ex87R02myV3noBKut7jUNCFOiQzXfOhH5247AIQrQzsZtTXVw3gSWRJBTP4YisUnC3X4asQ)](https://tipitaka.org/romn/cscd/tipitaka-latn.xsl)

**Comment by iulspop** [comment]
Hi @bksubhuti, why did you merge this PR when it was still WIP? I created a new PR to continue my work: https://github.com/bksubhuti/tpr_downloads/pull/3

**Comment by bksubhuti** [comment]
I was under the impression you could still keep editing this PR>> sorry.
I don't know PR stuff.  I just wanted to see the files and so I merged so I could download them.

**Comment by iulspop** [comment]
> I was under the impression you could still keep editing this PR>> sorry. I don't know PR stuff. I just wanted to see the files and so I merged so I could download them.

No worries @bksubhuti. You can view the files changed/added in the PR tab "files changed" and leave comments there as you review my code and documentation. If you wish to view the files locally, you can run the command `git checkout download-anna` in the project directory. It will pull the remote branch I'm working on to your local machine, then switch git to it. Once you're done viewing, you can switch back using `git checkout master`.

**Pull request merged**